core/src/test/scala/com/nvidia/spark/rapids/tool/tuning/ProfilingAutoTunerSuite.scala

```diff
@@ -3546,4 +3546,220 @@ class ProfilingAutoTunerSuite extends BaseAutoTunerSuite {
     initialValue = "128",
     expectedValue = Some("512m")
   )
+
+  test("Do not recommend RAPIDS Shuffle Manager when dynamic allocation is enabled") {
+    // mock the properties loaded from eventLog
```

Comment:
can we add a quick comment explaining what is the value of the unit test. If someone later change that method , then what is they need to watch for.
For example,
```
// this unit test evaluates that the AutoTuner set x.y.z to n when the configurations r.s.t is set to bla.bla.
```

Reply:
Sure, added.